src/torchmetrics/__init__.py

```diff
@@ -21,26 +21,19 @@
     SignalNoiseRatio,
 )
 from torchmetrics.classification import (  # noqa: E402
-    AUC,
     AUROC,
     ROC,
     Accuracy,
     AveragePrecision,
-    BinnedAveragePrecision,
-    BinnedPrecisionRecallCurve,
-    BinnedRecallAtFixedPrecision,
```

Comment:
not sure why these are removed...

Reply:
`AveragePrecision`, `PrecisionRecallCurve`, `RecallAtFixedPrecision` all support both binned and non-binned calculations and these specialized versions are therefore redundant

Reply:
good, let's do https://github.com/Lightning-AI/metrics/pull/1251#discussion_r992214449